mr/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -111,6 +111,10 @@
     if (schemaStr != null) {
       scan.project(SchemaParser.fromJson(schemaStr));
     }
+    String[] columnsToProject = conf.getStrings(InputFormatConfig.COLUMN_PROJECTIONS, null);
+    if (columnsToProject != null) {
+      scan.select(columnsToProject);
```

Comment:
In line with @rdblue's comment above, the usage of `columnsToProject` is potentially confusing given on the difference we've settled on for when to use `project` vs `select`. Additionally, there's a `project` call above on the parsed schema followed by this `select`, so the diffference between the two is especially noticeable here.

In addition to updating the added config's name to `SELECTED_COLUMNS`, can you possibly change `columnsToProject` to be `columnsToSelect` or even simpler, just `selectedColumns`?